ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-v0.21.0__47.yaml

```diff
@@ -153,10 +153,50 @@ images:
       - destination_dir: .
         source_path: /go/bin/recordevents
   to: knative-eventing-kafka-test-recordevents
+- dockerfile_path: openshift/ci-operator/knative-test-images/wathola-fetcher/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wathola-fetcher
+  to: knative-eventing-kafka-test-wathola-fetcher
+- dockerfile_path: openshift/ci-operator/knative-test-images/wathola-forwarder/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wathola-forwarder
+  to: knative-eventing-kafka-test-wathola-forwarder
+- dockerfile_path: openshift/ci-operator/knative-test-images/wathola-kafka-sender/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wathola-kafka-sender
+  to: knative-eventing-kafka-test-wathola-kafka-sender
+- dockerfile_path: openshift/ci-operator/knative-test-images/wathola-receiver/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wathola-receiver
+  to: knative-eventing-kafka-test-wathola-receiver
+- dockerfile_path: openshift/ci-operator/knative-test-images/wathola-sender/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wathola-sender
+  to: knative-eventing-kafka-test-wathola-sender
 promotion:
   additional_images:
     knative-eventing-kafka-src: src
-  name: knative-v0.21.1
+  name: knative-v0.21.0
```

Comment:
Is this okay, @matzew? You have mentioned it should be `v0.21.1`

Reply:
yeah - please revert this line - must be `knative-v0.21.1`